framework/src/errors.ts

```diff
@@ -73,3 +73,45 @@ export class ValidationError extends Error {
 		Error.captureStackTrace(this, FrameworkError);
 	}
 }
+
+export class SequenceModuleError extends Error {
```

Comment:
this should be defined within the module folder

Reply:
Removed this error